PublicMapping/districtbuilder
# Add "saved" notification

## Overview

Adds "Saved" notification to the sidebar, which appears when the map reflects what is saved on the server. This reinforces the connection between the "Accept" button and saving, and confirms to the user that their map is saved.

### Checklist


### Demo

![Screen Shot 2020-09-22 at 10 12 17 AM](https://user-images.githubusercontent.com/1809908/93896521-028d7700-fcbf-11ea-8e8a-7125bbd26b21.png)

### Notes

@tgilcrest I added tooltips for the "Saved" text as well as the "Cancel" and "Accept" buttons. Could you take a look at that and let me know if you have any edits.

![Screen Shot 2020-09-22 at 10 39 47 AM](https://user-images.githubusercontent.com/1809908/93897419-0a015000-fcc0-11ea-85b4-1e94263526dd.png)
![Screen Shot 2020-09-22 at 10 12 02 AM](https://user-images.githubusercontent.com/1809908/93897275-e3dbb000-fcbf-11ea-8c65-c2c28be6ffbd.png)
![Screen Shot 2020-09-22 at 10 11 58 AM](https://user-images.githubusercontent.com/1809908/93897277-e3dbb000-fcbf-11ea-8dc9-ac57e57ba7f5.png)

## Testing Instructions

- Open a project
- You should see a "Saved" indicator in the sidebar
- Make a selection on the map; this should remove the "Saved" indicator and show the "Cancel" and "Accept" buttons instead
- Press cancel; the "Saved" indicator should reappear
- Make a new selection on the map; this time click "Accept"; you should see a brief spinner and then the "Saved" indicator should return after the save is complete and the map updates

Closes #431
